### DISCUSSION TITLE
Discussion for improving editor simplicity

### DISCUSSION THREAD

**Discussion by n0x5** [WordPress/gutenberg#39159] [Ideas]
I hope it's ok to post several ideas in one thread.

I have some specific and some general.

1) So first off I think I understand why you need to press shift+enter to create a line break without a new paragraph, but I think this is unnatural and can lead to unintentionally creating a bunch of paragraph blocks. I think the key press should be inverted where Shift+enter creates a new paragraph block and enter creates a new line without a new block.
This would feel more natural and like you are in more control and I think users have to learn shift+enter at some point anyway.
There may be a use for a general "text block" that can contain paragraphs and headings and so on.

2) A "simple mode" in the editor.
A common task is to have a block of text and an image aligned next to each other. Another is to make a line of text bigger (create a heading) or to insert an image or a link. 
I think these common tasks should be always displayed as a menu underneath a block, and always visible, like "insert image next to text", "insert image under text", "insert a link", so that you can click on them and immediately get the result you want.
Typically now to even just align an image to the right of a block of text i just create a Columns block and then create a paragraph block in one column and an image block in the other but this is tedious. Very common tasks should be one click I think.
In general moving the creation and curation of specific blocks into the background and hiding them behind general tasks like mentioned above is good I think.

3) A "global settings" page where you can set background-color, font-size, link color of the entire site etc and then have those save to style.json. This way we can start with an empty theme and easily create a theme. Also some interface to add custom web fonts would be cool.

4) A "universal" query loop block. 
The query loop block is OK but it would be cool to have a universal one where you can select the table and columns directly in the interface and then have the block display those columns on the page. This would enable easy access to query the database and be almost the equivalent of a custom wp_query in Classic themes. I also image maybe a sub query section so that you can use the result of one column from root query to query another table (like postmeta), or, having built in sub queries for things like image meta data or custom taxonomies readily available to add if the root query supports it.
Edit: I forgot to add to this that I think each column should be an individual block automatically. That you can layout the blocks and set their width, color and all of this, like the Post Title or Post Content blogs, but for each column added.

Thanks for reading.

**Comment by annezazu** [comment]
Thanks for opening this discussion! I wanted to quickly connect some dots for you to some current initiatives/ideas being floated around that might interest you based on each of the above items:

>A "simple mode" in the editor.

A PR is in progress to drastically reduce the interface: https://github.com/WordPress/gutenberg/pull/38928 It's not quite what you're describing as "common tasks" are wildly different depending on each person's use case but it does show one pathway being considered. 

>A "global settings" page

From what you wrote, I think you'll like this idea of having a dedicated space for configuring global styles and settings for each block: https://github.com/WordPress/gutenberg/issues/32682 While it wasn't an initial current priority, like you, I can see having one place to edit and manage all of these options to be quite helpful. 

> A "universal" query loop block.

The idea of a Query Loop builder has been tossed around. You can see the designs and idea in more detail here:  https://github.com/WordPress/gutenberg/discussions/35760 Might be neat to chime in there.

Hope this helps :)

**Reply by n0x5** [reply]
Thanks a lot, I'm reading up on these.
